it/controllers/AgentsControllerISpec.scala

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import models.NormalMode
+import play.api.http.Status
+import play.api.libs.json.Json
+import play.api.mvc.AnyContent
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import stubs.AuthStub
+import utils.{IntegrationSpecCommonBase, SessionKeys}
+
+import java.time.LocalDateTime
+
+class AgentsControllerISpec extends IntegrationSpecCommonBase {
+  val controller: AgentsController = injector.instanceOf[AgentsController]
+
+  "GET /who-planned-to-submit-vat-return" should {
+    "return 200 (OK) when the user is authorised" in {
+      val fakeRequestWithCorrectKeys: FakeRequest[AnyContent] = FakeRequest("GET", "/why-return-was-submitted-late-agent").withSession(
```

Comment:
Should this be "/who-planned-to-submit-vat-return"?

Reply:
Done